### PR TITLE
Added Console Log Listener Test

### PR DESCRIPTION
**Pull request by ins0**
This PR adds a TestCase for the Console Listener.
It's not the best approach but as the class uses ``new Classname`` we have no choice.
Still better than no test.

This PR:

- [x] adds a console test
- [x] fixes EOL on different systems

### CODE REVIEW

**Comment by frickelbruder** [comment]
Hi @ins0 ,

thanks for your work so far. You are totally right to have a Unit Test on this rather important part of the tool.
But I think, it would be better to be able to inject either the $consoleOutput in the constructor, or set it automatically as it is now. This gives us the opportunity to inject a proper interface and implement our stub against the interface, not against the implementation inside the class.

I do not like the reflection approach with unit tests at all, since the test has to much knowledge about the tested class and I cannot be sure anymore to not break anything, if I change private members of a class.

Do you still want to work on this?

Grüße
Markus :-)

**Comment by ins0** [comment]
Used the reflection only because i thaught about the test in the first place - rather then to touch multiple files for this. Sure.

**Comment by ins0** [comment]
@frickelbruder done ;)

File: src/Log/Listener/ConsoleOutputListener.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class ConsoleOutputListener implements Listener {
      */
     protected $consoleOutput;
 
-    protected $outputPadLength = 80;
+    protected $outputPadLength = 0;
```
Comment (ins0) [inline src/Log/Listener/ConsoleOutputListener.php]: @frickelbruder not sure why ``$outputPadMinLength`` was there in the first place (unused)

File: src/Log/Listener/CsvLogListener.php [hunk 2]
```diff
@@ -13,16 +13,12 @@ public function log($sectionName, $targetUrl, RuleInterface $rule, $success) {
         $this->logs[] = array($sectionName, $targetUrl, $rule->getName(), $success ? 'Ok' : 'FAILED');
     }
 
-
     public function finish() {
         $handle = fopen($this->logFileName, 'w');
         foreach($this->logs as $log) {
-            fputcsv($handle, $log);
+            fwrite($handle, implode(',', $log) . PHP_EOL);
```
Comment (frickelbruder) [inline src/Log/Listener/CsvLogListener.php]: Why did you change this?
fputcsv does a little more than turning an array into comma seperated lists. It also adds quotes where needed.

**Comment by frickelbruder** [comment]
Ok,
I'll merge this PR, as I think it is good as it is.

I don't like the fwrite substitution and will revert it back. There is another fix for this in the 2.0 branch, which does not change the code but is a fix in the tests, where the EOL-thing is mostly annoying.

One thing:
It would be better to provide one encapsuled feature per PR. This makes it easier to track and/or merge the changes.

But many thanks anyway :-)
Regards
Markus
